lib/features/welcome/views/widgets/welcome_button.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:kahootify/color_consts.dart';
+
+class WelcomeButton extends StatelessWidget {
+  final Function onPressed;
+  final String value;
```

Comment:
the parameter name value doesn't feel right for what it is representing. It's not value of a button, it's rather it's text or name. Refactor it!

Reply:
Ready